int32 top
int32 left
int32 right
int32 bottom
string name
float32 confidence
